common/openapi.yml

```
@@ -236,6 +236,7 @@ components:
 
     DatasetsListResponse:
       type: object
+      required: [result]
```

```suggestion
      required: [ result ]
```
suggestion: add spaces in the array, since that's the style the rest of the code is in